Modules/Filtering/FFT/include/itkFFTPadImageFilter.h

```diff
@@ -82,10 +82,12 @@ class ITK_TEMPLATE_EXPORT FFTPadImageFilter : public PadImageFilterBase<TInputIm
 
   /**
    * Set/Get the greatest prime factor allowed on the size of the padded image.
+   *
    * The filter increase the size of the image to reach a size with the greatest
-   * prime factor smaller or equal to the specified value. The default value is
-   * 13, which is the greatest prime number for which the FFT are precomputed
-   * in FFTW, and thus gives very good performance.
+   * prime factor smaller or equal to the specified value. The default values if
```

Comment:
default values if -> default value is

on the previous line increase -> increases